projects/igniteui-angular/src/lib/tabs/tab-content.directive.ts

```diff
@@ -0,0 +1,32 @@
+import { Directive, ElementRef, HostBinding } from '@angular/core';
+import { IgxTabItemDirective } from './tab-item.directive';
+import { IgxTabContentBase } from './tabs.base';
+
+@Directive()
+export abstract class IgxTabContentDirective implements IgxTabContentBase {
+
+    /** @hidden */
+    @HostBinding('attr.role')
+    public role = 'tabpanel';
+
+    /** @hidden */
+    constructor(public tab: IgxTabItemDirective, private elementRef: ElementRef) {

```

Comment:
```suggestion
    constructor(public tab: IgxTabItemDirective, private elementRef: ElementRef<HTMLElement>) {
```
I think this is cleaner as you won't have to declare a return type for the `nativeElement` getter.